DataManagement/GitTutorial.Rmd

```diff
@@ -196,60 +218,100 @@ git clone git:kingsleyshacklebolt/dragon_study.git
 ```
 <br/><br/>
 Restart Rstudio.
-If it worked you can see the git panel with a green and blue arrow to push and pull.
+You should now see the git panel with a green and blue arrow to push and pull.
 
-!!! ADD FIGURE OF GIT PANEL!!!
+<br/><br/>
+```{r git_window}
+knitr::include_graphics("Pics/gittutorial/17_git_window.png")
```

Comment:
on an existing project, this would be full of yellow files

Reply:
Good point